src/glossary.rst

```diff
@@ -1216,102 +1449,133 @@ Glossary
+      A flow is a single logical run of a :term:`workflow` that follows the
+      :term:`graph`, or part of the graph, onward from some start point.
 
-      Reflows are useful when you need to re-wind your :term:`workflow` run to
-      allow it to evolve a new path into the future.
+      Cylc :term:`schedulers <scheduler>` can manage more than one flow in a
+      single graph, at the same time. We call this capability *reflow*.
+      The original flow in a new workflow run is :term:`started <startup>`
+      with the ``cylc play`` command. Subsequent flows may be triggered from
+      particular task(s) in the graph using ``cylc trigger --reflow``.
 
-      .. TODO - add this once the CLI example works
+      Flows are identified by a :term:`flow number` that starts at ``1`` and
+      increments for each triggered flow.
 
-         For example, the following workflow runs the task ``a`` every cycle, then
-         either ``b`` or ``c`` depending on the outcome of ``a``:
+      Reflows are useful if you need to re-wind your :term:`workflow` to allow
+      it to evolve a new path into the future, or to repeat-run some part of
+      the graph to regenerate its outputs after making changes.
 
-         .. code-block:: cylc
+      For example, the following :term:`cycling workflow` runs a :term:`task`
+      called ``model`` in every cycle, followed by a postprocessing task, two
+      product generating tasks, and finally a task that publishes results for
+      the cycle point:
 
-            [scheduler]
-                allow implicit tasks = True
+         .. code-block:: cylc
 
             [scheduling]
                 cycling mode = integer
                 initial cycle point = 1
                 [[graph]]
                     P1 = """
-                        b[-P1] | c[-P1] => a
-                        a:succeed => b
-                        a:fail => c
+                        model [-P1] => model => post => prod1 & prod2 => publish
                     """
 
-         Let's say the workflow has run to cycle 8, but we have just noticed that
-         something went wrong with the task ``a`` in cycle 5.
+      Let's say the workflow has run to cycle 8, but we have just noticed that
+      a corrupted ancillary file resulted in bad products at cycle 5.
 
-         To rectify this we could create a new flow (a reflow) starting at ``a.5``::
+      To rectify this we could fix the corrupted file and trigger a new flow
+      (a reflow) from ``proc.5``::
 
-            # get the ID of the original flow
-            cylc TBC <id> a.5  # ???
+         cylc trigger a.5 --reflow <workflow-id>
 
-            # pause the original flow
-            cylc pause <id> --flow <flow>
+      The new flow will regenerate and republish the cycle 5 products before
+      naturally coming to a halt because the the triggered tasks do not lead on
+      to the next cycle and the rest of the graph.
```

Comment:
I don't quite get this. As there is `model[-P1] => model`, why doesn't cycle 6 start running?

Reply:
Damn it _2 ... I had also fixed this, and moved it to a new section on relow.

To answer your question, the example is supposed to trigger the postprocessing task, not the model.